README.md

```diff
@@ -1,6 +1,6 @@
 # DOT Birthday Countdown
 
-The purpose of this project is to help members of DOT establish their skills by co-creating on a birthday countdown of significant figures.
+The purpose of this project is to be a birthday countdown of influential public persons.
```

Comment:
Even though a majority of the people will probably end up being public figures, I wouldn't officially limit it to that. We can include ourselves or historical figures from the past. In regards to the countdown, the importance of famous people is to verify the date and to show the countdown works. The programming logic behind this countdown demo wont be dependent on the popularity of the person.

The purpose of the demo is to show a working example using "ReactJS", in this case, of a countdown to a person next birthday. It can probably be worded a little bit better than I just explained it. Once this is demo is stable, we could create another repo that does the same exact thing but using Angular or another Framework/Library. It would be great for Educational reasons to see the difference in implementation.